package org.stepik.week4;

/**
 * @author Nika Zurbaevi
 */
public class QuestionAnswer_4_2_6 {
    /*
     * Вопрос: Рассмотрим пример кода:
     *
     *     try {
     *       //
     *     } catch (Exception e) {
     *       // ...
     *     } finally {
     *       // ...
     *     }
     *
     * При каких условиях блок finally не будет выполнен? Можно выбрать несколько вариантов.
     *
     *
     * Ответ:
     * 1) Если во время исполнения блока try виртуальная машина "выполнила недопустимую операцию и будет закрыта".
     * 2) Если в блоке try вызван System.exit(0).
     *
     */
}
